index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <header>
+        <!-- <p>My Logo</p> -->
+        <div>
+            <span>My Logo</span>
+            <!-- <img class="navbar" src="images/menu.png" alt="navigation"> -->
+            <img src="images/menu.png" alt="navigation">
+        </div>
+    </header>
+    <section>
+        <h1>I'm Norman<br> Glad to see you!</h1>
+        <p>I’m a software developer! I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p class="connect">LET'S CONNECT</p>
+        <div class="socials">
+            <img src="images/1.png" alt="">
+            <img src="images/2.png" alt="">
+            <img src="images/3.png" alt="">
+            <img src="images/4.png" alt="">
+            <img src="images/5.png" alt="">
+        </div>

```

Comment:
- [ ] Since this is a list, it is semantically more accurate to use `<ul> `tag. Besides, these are links to your social media profiles such as `LinkedIn, Angelist, Github,` etc. Therefore the `<img>` should be wrapped in an `<a>` tag and the `href` can be assigned your social media profile links.
```
<ul>
  <li> Ma href=""><img src=""></a></li>
  ....
</ul>
```